src/legendhpges/p00664b.py

```diff
@@ -56,13 +56,17 @@ def _tan(a):
 
         r = []
         z = []
+        surfaces = []
 
         if c.pp_contact.depth_in_mm > 0:
             r += [0, c.pp_contact.radius_in_mm, c.pp_contact.radius_in_mm]
             z += [c.pp_contact.depth_in_mm, c.pp_contact.depth_in_mm, 0]
+            surfaces += ["p+", "passive"]
```

Comment:
I'm being picky here but the insulating surface is not always passivated, maybe use `groove` or `insulation` or whatever?

Reply:
ok i chose passive to be inclusive of ppcs (ie not groove)